content/efcore/getting-started/loading-related-data.md

```diff
@@ -154,18 +154,18 @@ using (var context = new LibraryContext())
 :::repl{data-name=count-method} 
 :::
  
-Likewise, we can find the year of the oldest edition of a book using the `Min` method: 
+Likewise, we can find the date of the first checkout record of a book using the `Min` method: 
```

Comment:
~~...date of the first time a specific book was checked out using...~~

Actually, could we switch this to "Likewise, we can find the most recent date a specific book was checked out using the `Max` method:"